docs/README.build.md

```diff
@@ -0,0 +1,24 @@
+# How to build snabb-lwaftr
+
+## Step 1: Fetch the sources
+
+```bash
+$ git clone https://github.com/Igalia/snabbswitch.git
+```
+
+## Step 2: Check out the lwaftr development branch:
+
+```bash
+$ cd snabbswitch && git checkout lwaftr_nutmeg
+```
+
+## Step 3: Build
+
+```bash
+$ make
```

Comment:
Thanks @lukego , we're moving towards something more like that system. I mainly like that model.